pvacseq/lib/binding_filter.py

```diff
@@ -7,8 +7,6 @@
 
 def main(args_input = sys.argv[1:]):
     parser = argparse.ArgumentParser('pvacseq binding_filter')
-    parser.add_argument('input', type=argparse.FileType('r'),
-                        help="Input list of variants")
     parser.add_argument('fof', type=argparse.FileType('r'),
```

Comment:
Not really related to this PR, per se, but I wonder if it would be easier to just ask the user to specify a list of file paths in the input instead of a path to a fof. I think it would make the interface across the board easier since, e.g. the main.py wouldn't need to write a fof. @jhundal @jasonwalker80 what do you guys think? If we want to change this functionality then now is the time to do it since we're changing the API in other places as well. Since this change is outside of the scope of this PR I would suggest a separate PR for it.